### PR TITLE
Fix `check_is_pair` for SRA-toolkit's format

### PR DESCRIPTION
**Pull request by kdm9**
See comments here:

https://github.com/dib-lab/khmer/pull/1088/files#r34636674

### CODE REVIEW

**Comment by ctb** [comment]
@kdmurray91 I think I would like to see a test that breaks on the old code, too, to make sure that the lhs bug doesn't creep back in later. I'm happy to contribute such a thing if you like?

**Comment by kdmurray91** [comment]
OK, go ahead. Was going to hack one of the original tests to do so. Note this PR isn't finished :)

**Comment by ctb** [comment]
oh, ok, if you can do it then I can merge - otherwise we'll need to find a
3rd reviewer :)

On Thu, Jul 16, 2015 at 06:10:24AM -0700, Kevin Murray wrote:
> OK, go ahead. Was going to hack one of the original tests to do so. Note this PR isn't finished :)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dib-lab/khmer/pull/1173#issuecomment-121951452
-- 
C. Titus Brown, ctbrown@ucdavis.edu

**Comment by mr-c** [comment]
@ctb I was told by @kdmurray91 that he'll be offline for a few days; I am happy to be the 3rd party to your contribution

**Comment by ctb** [comment]
@mr-c ready for review.

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2015-07-23  Titus Brown  <t@titus-air-2.attlocal.net>
```
Comment (mr-c) [inline ChangeLog]: Is that the email address you want to use?

**Comment by mr-c** [comment]
@ctb checklist?

**Comment by ctb** [comment]
Whoops, thanks @mr-c!

- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

**Comment by mr-c** [comment]
LGTM, merged

**Pull request merged**